validator/src/main.rs

```diff
@@ -509,6 +509,12 @@ pub fn main() {
                 .takes_value(false)
                 .help("drop older slots in the ledger"),
         )
+        .arg(
+            clap::Arg::with_name("force_compaction")
```

Comment:
Once we're comfortable with this change, why would we not enable it always?   Or at least flip the default: `--no-force-ledger-compaction`

Reply:
:+1:, force compaction is now implied by `limit-ledger-size`